apps/meteor/app/videobridge/client/tabBar.tsx

```diff
@@ -57,7 +58,7 @@ addAction('start-call', ({ room }) => {
 	const live = room?.streamingOptions && room.streamingOptions.type === 'call';
 	const enabled = enabledDMs || enabledChannel || enabledTeams || enabledGroups || enabledLiveChat;
 
-	const enableOption = enabled && (!user?.username || !room.muted?.includes(user.username));
+	const enableOption = enabled && (!user?.username || !room.muted?.includes(user.username)) && canStartCall;
```

Comment:
nit: can avoid some computation this way
```suggestion
	const enableOption = enabled && canStartCall && (!user?.username || !room.muted?.includes(user.username));
```